Assets/VRTK/Scripts/VRTK_MoveInPlace.cs

```diff
@@ -0,0 +1,387 @@
+﻿// Move In Place|Scripts|0145
+namespace VRTK
+{
+    using UnityEngine;
+    using System.Collections.Generic;
+
+    /// <summary>
+    /// Move In Place allows you to move the play area by calculating the y-movement of your controllers and propelling you the more they are moving.  This simulates walking in game by walking in real life.
+    /// </summary>
+    /// <remarks>
+    ///   >This locomotion method is based on Immersive Movement, originally created by Highsight.
+    /// It is recommend you combine this with the VRTK_HeightAdjustTeleport
```

Comment:
Again, refrain from `you`

something like `It is recommended that the VRTK_HeightAdjustTeleport script is used in conjunction with this script`